seumich/views.py

```diff
@@ -8,18 +8,18 @@
 from django.core.exceptions import ObjectDoesNotExist, MultipleObjectsReturned
 from django.db.models import Q, Prefetch, Value as V
 from django.db.models.functions import Concat
+from django.contrib.auth import get_user_model
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.contrib import messages
-from django.conf import settings
 from tracking.utils import UserLogPageViewMixin
 
 import operator
 import logging
 
-from django.conf import settings
 from decouple import config
 from functools import reduce
 
+User = get_user_model()
```

Comment:
You only use this once, you can probably just call it directly on line 98 as
get_user_model().objects.get

I'd at least move it out of the global module namespace.

Reply:
Done. See [this commit](https://github.com/tl-its-umich-edu/student_explorer/pull/161/commits/8d66ae1ba1c83b33642b68ecc2b8393396587da1).